src/aks-preview/azext_aks_preview/addonconfiguration.py

```diff
@@ -112,6 +116,7 @@ def enable_addons(cmd,
                 instance.location,
                 aad_route=False,
                 enable_syslog=enable_syslog,
+                data_collection_settings=data_collection_settings
```

Comment:
Currently aks-preview relies on azure-cli of version [2.43.0](https://github.com/Azure/azure-cli-extensions/blob/main/src/aks-preview/azext_aks_preview/azext_metadata.json), and that version of azure-cli does not yet contain [the change](https://github.com/Azure/azure-cli/pull/24777) (should be included in the upcoming [2.44.0](https://github.com/Azure/azure-cli/milestone/125) release) to be reused.

So either wait for azure-cli to complete the release and aks-preview to update the dependencies before making changes, or copy the code from the azure-cli side.

Reply:
> Currently aks-preview relies on azure-cli of version [2.43.0](https://github.com/Azure/azure-cli-extensions/blob/main/src/aks-preview/azext_aks_preview/azext_metadata.json), and that version of azure-cli does not yet contain [the change](https://github.com/Azure/azure-cli/pull/24777) (should be included in the upcoming [2.44.0](https://github.com/Azure/azure-cli/milestone/125) release) to be reused.
> 
> So either wait for azure-cli to complete the release and aks-preview to update the dependencies before making changes, or copy the code from the azure-cli side.

> Sure, will wait for the aks-preview to get the updated for 2.44.0.  The official release date of 2.44.0 is 1/10/23, what's the ETA for aks-preview to get updated for this version?

Reply:
I'll bump the dependency soon after the release of azure-cli, eta 01/13.

Reply:
Hi, @FumingZhang  - can you please approve and merge this PR after bumping the dependency?

Reply:
Hey @ganga1980, opened this PR [#5759](https://github.com/Azure/azure-cli-extensions/pull/5759) to bump the dependency.

Reply:
> Hey @ganga1980, opened this PR [#5759](https://github.com/Azure/azure-cli-extensions/pull/5759) to bump the dependency.

> @FumingZhang ,  I see this PR https://github.com/Azure/azure-cli-extensions/pull/5759 already. can you please review and merge this PR as well?

Reply:
Well, there are some conflicts in your PR. Also it seems something is missing when syncing the change to aks-preview. I've opened this new [PR #5779](https://github.com/Azure/azure-cli-extensions/pull/5779) based on your changes. PTAL

Reply:
Hi, @FumingZhang - Thanks for  resolving conflicts and making PR. Changes in the PR https://github.com/Azure/azure-cli-extensions/pull/5779, looks good to me. Once you merged that PR, please close this PR.

Reply:
Sure